go/worker/storage/committee/node.go

```diff
@@ -226,11 +226,17 @@ func NewNode(
 				return nil, fmt.Errorf("failed to retrieve genesis block: %w", rerr)
 			}
 
+			var threads uint16
+			if config.GlobalConfig.Storage.Checkpointer.ParallelChunker {
+				threads = 12
```

Comment:
Move this into a constant, with some short reasoning/explanation for the picked value.